### ISSUE TITLE
plot_dimred throws "Column `arrow` must be length 1"

### ISSUE THREAD

**Issue by phitlingt** [dynverse/dynplot#41]
This error does not happen often with dataset generated by dyntoy.
About 3 out of 10 times I tried running following code I get error "Column `arrow` must be length 1".

```
library(dyntoy)
toy_data <- dyntoy::generate_dataset(
   id = "specific_example/paga",
   num_cells = 1000,
   num_features = 101,
   model = "tree",
   normalise = FALSE
 )
toy_model <- dynwrap::infer_trajectory(toy_data, ti_paga(n_neighbors=15L, n_comps=50L, 
n_dcs=15L, resolution=1L, embedding_type="umap", connectivity_cutoff=0.05), verbose=TRUE)
plot_dimred(toy_model)
```
> Error: Column `arrow` must be length 1 (the group size), not 0

However, I constantly run into this error when I am analyzing Setty 2019 dataset from [Palantir paper](https://www.nature.com/articles/s41587-019-0068-4#Sec14).
[Here](https://github.com/phitlingt/DATA/blob/master/datasetInput_Setty2019.RData) is the file to reproduce the issue.
```
dataset <- wrap_expression(
  counts=t(hvg.sce.cts),
  expression=t(hvg.sce.logcts)
)

dataset <- add_prior_information(
  dataset,
  start_id="Run5_164698952452459"
)

model <- dynwrap::infer_trajectory(dataset, ti_paga(n_neighbors=15L, n_comps=50L, n_dcs=15L,
         resolution=1L, connectivity_cutoff=0.05), give_priors=c("start_id"), verbose=TRUE)

plot_dimred(model)
```

**Comment by lixin4306ren** [comment]
Got the same error using my own data. Any suggestion?

**Comment by rcannood** [comment]
Hello @phitlingt and @lixin4306ren,

Sorry for the delayed response. Your problem should be fixed in dynplot 1.0.2. Could you check to make sure this is indeed the case?

Robrecht

**Comment by phitlingt** [comment]
Hi,

Apologize for the delay in response.

I have actually been using paga / paga_tree a lot,
and I have not encountered this issue since the fix in July.

Thank you very much for the wonderful work again!
